trust-quorum/src/coordinator_state.rs

```diff
@@ -101,18 +106,21 @@ impl CoordinatorState {
             new_shares,
         };
 
-        Ok(CoordinatorState::new(now, msg, config, op))
+        Ok(CoordinatorState::new(log, now, msg, config, op))
     }
 
     // Intentionallly private!
```

Comment:
```suggestion
    // Intentionally private!
```

I guess this is because the public constructors do error checking? If so could this comment briefly mention that?

Reply:
Done in [50ebf64](https://github.com/oxidecomputer/omicron/pull/8000/commits/50ebf6479db5b3e8e345394a1bcf2943306edf78)